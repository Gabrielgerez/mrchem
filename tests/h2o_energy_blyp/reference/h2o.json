{
  "input": {
    "constants": {
      "N_a": 6.02214076e+23,
      "angstrom2bohrs": 1.8897261246257702,
      "boltzmann_constant": 1.380649e-23,
      "dipmom_au2debye": 2.5417464739297717,
      "e0": 8.8541878128e-12,
      "electron_g_factor": -2.00231930436256,
      "elementary_charge": 1.602176634e-19,
      "fine_structure_constant": 0.0072973525693,
      "hartree2ev": 27.211386245988,
      "hartree2kcalmol": 627.5094740630558,
      "hartree2kjmol": 2625.4996394798254,
      "hartree2simagnetizability": 78.9451185,
      "hartree2wavenumbers": 219474.6313632,
      "light_speed": 137.035999084,
      "meter2bohr": 18897261246.2577
    },
    "geom_opt": {
      "init_step_size": -0.5,
      "max_force_component": 0.005,
      "max_history_length": 10,
      "max_iter": 100,
      "minimal_step_size": 0.01,
      "run": false,
      "subspace_tolerance": 0.001,
      "use_previous_guess": false
    },
    "molecule": {
      "charge": 0,
      "coords": [
        {
          "atom": "o",
          "r_rms": 5.0580178957e-05,
          "xyz": [
            0.0,
            0.0,
            -0.12868309568322003
          ]
        },
        {
          "atom": "h",
          "r_rms": 2.6569547399e-05,
          "xyz": [
            -1.4375,
            0.0,
            1.0213169043167798
          ]
        },
        {
          "atom": "h",
          "r_rms": 2.6569547399e-05,
          "xyz": [
            1.4375,
            0.0,
            1.0213169043167798
          ]
        }
      ],
      "multiplicity": 1
    },
    "mpi": {
      "bank_per_node": -1,
      "bank_size": -1,
      "numerically_exact": true,
      "omp_threads": -1,
      "shared_memory_size": 10000,
      "use_omp_num_threads": false
    },
    "mra": {
      "basis_order": 6,
      "basis_type": "interpolating",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -5
    },
    "printer": {
      "file_name": "h2o",
      "print_constants": false,
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {},
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 0.001,
          "shared_memory": false
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 0.001
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "nuclear_model": "point_like",
          "proj_prec": 0.001,
          "shared_memory": false,
          "smooth_prec": 0.001
        },
        "xc_operator": {
          "shared_memory": false,
          "xc_functional": {
            "cutoff": 0.0,
            "functionals": [
              {
                "coef": 1.0,
                "name": "blyp"
              }
            ],
            "spin": false
          }
        }
      },
      "initial_guess": {
        "environment": "None",
        "external_field": "None",
        "file_CUBE_a": "cube_vectors/CUBE_a_vector.json",
        "file_CUBE_b": "cube_vectors/CUBE_b_vector.json",
        "file_CUBE_p": "cube_vectors/CUBE_p_vector.json",
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": true,
        "method": "DFT",
        "prec": 0.001,
        "relativity": "None",
        "restricted": true,
        "rotate": true,
        "screen": 16.0,
        "type": "gto",
        "zeta": 0
      },
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 0.001,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        },
        "quadrupole_moment": {
          "quad-1": {
            "operator": "h_e_quad",
            "precision": 0.001,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        }
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        4.6220056999883845e-18,
        0.0,
        -2.863104619580819e-17
      ],
      "charge": 0,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 0.7424565698413224,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            7.541414894830454e-07,
            -8.818233113258066e-12,
            0.7424565698409393
          ],
          "vector_el": [
            7.541484277108155e-07,
            0.0,
            -0.27071247333314297
          ],
          "vector_nuc": [
            -6.938227770092453e-12,
            -8.82056458160978e-12,
            1.0131690431740823
          ]
        }
      },
      "geometry": [
        {
          "symbol": "O",
          "xyz": [
            0.0,
            0.0,
            -0.12868309568322003
          ]
        },
        {
          "symbol": "H",
          "xyz": [
            -1.4375,
            0.0,
            1.0213169043167798
          ]
        },
        {
          "symbol": "H",
          "xyz": [
            1.4375,
            0.0,
            1.0213169043167798
          ]
        }
      ],
      "multiplicity": 1,
      "orbital_energies": {
        "energy": [
          -18.75208068966063,
          -0.8875930422750062,
          -0.4455912653784512,
          -0.30835763749397654,
          -0.22266530651236002
        ],
        "occupation": [
          2.0,
          2.0,
          2.0,
          2.0,
          2.0
        ],
        "spin": [
          "p",
          "p",
          "p",
          "p",
          "p"
        ],
        "sum_occupied": -41.23257588264085
      },
      "quadrupole_moment": {
        "quad-1": {
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "tensor": [
            1.5140299912491209,
            0.0,
            6.386018775780519e-07,
            0.0,
            -1.5103855436039533,
            -1.3882393656123874e-12,
            6.386018775780519e-07,
            -1.3882393656123874e-12,
            -0.0036444476451653884
          ],
          "tensor_el": [
            -1.5094569332683818,
            0.0,
            6.385991684118272e-07,
            0.0,
            1.6653462819448817,
            0.0,
            6.385991684118272e-07,
            0.0,
            -0.1558893486764977
          ],
          "tensor_nuc": [
            3.0234869245175027,
            0.0,
            2.709166224690307e-12,
            0.0,
            -3.175731825548835,
            -1.4013399973029642e-12,
            2.709166224690307e-12,
            -1.4013399973029642e-12,
            0.1522449010313323
          ]
        }
      },
      "scf_energy": {
        "E_ee": 46.76220073735242,
        "E_eext": 0.0,
        "E_el": -85.37030481026045,
        "E_en": -198.66012904564005,
        "E_kin": 75.84657365457292,
        "E_next": 0.0,
        "E_nn": 9.039235444235468,
        "E_nuc": 9.039235444235468,
        "E_tot": -76.33106936602498,
        "E_x": 0.0,
        "E_xc": -9.318950156545752,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 1,
      "nthreads": 8,
      "routine": "mrchem.x",
      "total_cores": 8,
      "version": "1.2.0-alpha"
    },
    "rsp_calculations": null,
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 46.76220073735242,
        "E_eext": 0.0,
        "E_el": -85.37030481026045,
        "E_en": -198.66012904564005,
        "E_kin": 75.84657365457292,
        "E_next": 0.0,
        "E_nn": 9.039235444235468,
        "E_nuc": 9.039235444235468,
        "E_tot": -76.33106936602498,
        "E_x": 0.0,
        "E_xc": -9.318950156545752,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": true
  }
}
