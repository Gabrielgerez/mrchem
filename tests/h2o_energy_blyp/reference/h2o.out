                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.2.0-alpha                              ***
***                                                                     ***
***         Git branch         text                                     ***
***         Git commit hash    20e53b0ebebbbed503cb-dirty               ***
***         Git commit author  gitpeterwind                             ***
***         Git commit date    Thu Feb 20 16:31:25 2025 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :       (no bank)                               1
 OpenMP threads          :                                               8
 Total cores             :                                               8
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.6.0-alpha
 Git branch            : merge
 Git commit hash       : d82d0d16fbefc341d12f
 Git commit author     : gitpeterwind
 Git commit date       : Thu Feb 20 16:02:13 2025 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (8 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 6
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       O            :        0.000000        0.000000       -0.128683
    1       H            :       -1.437500        0.000000        1.021317
    2       H            :        1.437500        0.000000        1.021317
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000       -0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                Computing Initial Guess Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Project GTO molecular orbitals
 Precision               : 1.00000e-03
 Screening               : 1.60000e+01 StdDev
 Restricted              : True
 MO file                 : initial_guess/mrchem.mop
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               5
 Beta electrons          :                                               5
 Total electrons         :                                              10
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    2    p          :                              1.000000203320e+00
    1    2    p          :                              1.000000064900e+00
    2    2    p          :                              9.999999960131e-01
    3    2    p          :                              1.000000004599e+00
    4    2    p          :                              9.999999947944e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT
 Relativity              : None
 Environment             : None
 External fields         : None
 Precision               : 1.00000e-03
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                 75.846573654573
 E-N energy              :            (au)               -198.660129045640
 Coulomb energy          :            (au)                 46.762200737352
 Exchange energy         :            (au)                  0.000000000000
 X-C energy              :            (au)                 -9.318950156546
 N-N energy              :            (au)                  9.039235444235
---------------------------------------------------------------------------
 Electronic energy       :            (au)                -85.370304810260
 Nuclear energy          :            (au)                  9.039235444235
---------------------------------------------------------------------------
 Total energy            :            (au)             -7.633106936602e+01
                         :      (kcal/mol)             -4.789846919254e+04
                         :        (kJ/mol)             -2.004071951016e+05
                         :            (eV)             -2.077074211088e+03
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                -18.752080689661
    1    2    p          :            (au)                 -0.887593042275
    2    2    p          :            (au)                 -0.445591265378
    3    2    p          :            (au)                 -0.308357637494
    4    2    p          :            (au)                 -0.222665306512
---------------------------------------------------------------------------
 Sum occupied            :            (au)                -41.232575882641
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       O            :        0.000000        0.000000       -0.128683
    1       H            :       -1.437500        0.000000        1.021317
    2       H            :        1.437500        0.000000        1.021317
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000       -0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                 75.846573654573
 E-N energy              :            (au)               -198.660129045640
 Coulomb energy          :            (au)                 46.762200737352
 Exchange energy         :            (au)                  0.000000000000
 X-C energy              :            (au)                 -9.318950156546
 N-N energy              :            (au)                  9.039235444235
---------------------------------------------------------------------------
 Electronic energy       :            (au)                -85.370304810260
 Nuclear energy          :            (au)                  9.039235444235
---------------------------------------------------------------------------
 Total energy            :            (au)             -7.633106936602e+01
                         :      (kcal/mol)             -4.789846919254e+04
                         :        (kJ/mol)             -2.004071951016e+05
                         :            (eV)             -2.077074211088e+03
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                -18.752080689661
    1    2    p          :            (au)                 -0.887593042275
    2    2    p          :            (au)                 -0.445591265378
    3    2    p          :            (au)                 -0.308357637494
    4    2    p          :            (au)                 -0.222665306512
---------------------------------------------------------------------------
 Sum occupied            :            (au)                -41.232575882641
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :        0.000001        0.000000       -0.270712
 Magnitude               :            (au)                        0.270712
                         :         (Debye)                        0.688082
---------------------------------------------------------------------------
 Nuclear vector          :       -0.000000       -0.000000        1.013169
 Magnitude               :            (au)                        1.013169
                         :         (Debye)                        2.575219
---------------------------------------------------------------------------
 Total vector            :        0.000001       -0.000000        0.742457
 Magnitude               :            (au)                        0.742457
                         :         (Debye)                        1.887136
===========================================================================


===========================================================================
                        Quadrupole Moment (quad-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic tensor       :       -1.509457        0.000000        0.000001
                         :        0.000000        1.665346        0.000000
                         :        0.000001        0.000000       -0.155889
---------------------------------------------------------------------------
 Nuclear tensor          :        3.023487        0.000000        0.000000
                         :        0.000000       -3.175732       -0.000000
                         :        0.000000       -0.000000        0.152245
---------------------------------------------------------------------------
 Total tensor            :        1.514030        0.000000        0.000001
                         :        0.000000       -1.510386       -0.000000
                         :        0.000001       -0.000000       -0.003644
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h  0m  7s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
